Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -307,21 +306,14 @@ YUI.add('ez-locationviewviewservice', function (Y) {
                 });
 
                 endLoadPath = tasks.add();
-                discoveryService.getInfoObject('rootLocation', function (error, response) {
-                    var rootLocationId;
-
+                location.loadPath(loadOptions, function(error, response) {
```

Comment:
nitpick: in this context, `response` is a bad name, the variable should be called `locations` or `path`.